### PR TITLE
Update delete TAModule command

### PR DESCRIPTION
**Pull request by EvaderFati**
- When deleting a TA module, delete all the class groups that tied to it.

### CODE REVIEW

**Review by lkldev — APPROVED** [review]
Thanks for the fix! LGTM 👍

File: src/main/java/seedu/address/model/ModelManager.java [hunk 1]
```diff
@@ -121,6 +123,10 @@ public void deleteEntity(Entity target) {
             tAssist.removeStudent(EntityConverter.entityToStudent(target));
             break;
         case TA_MODULE:
+            // remove all the class groups tied to the current module
```
Comment (Gernene) [inline src/main/java/seedu/address/model/ModelManager.java:126]: Super nitpick, but we might want to consider abstracting this later down the line. Other than this, looks good!

**Review by Gernene — APPROVED** [review]
Left one note. Other than that, looks good!

File: src/main/java/seedu/address/model/ModelManager.java (hunk 1 above)
Comment (lkldev) [reply src/main/java/seedu/address/model/ModelManager.java]: resolved in latest commit 👍

**Pull request merged**